plat/kvm/Config.uk

```diff
@@ -63,6 +63,19 @@ config KVM_BOOT_EFI_STUB_DTB_FNAME
 	string "Name of the Devicetree Blob file"
 	default "$(UK_NAME).dtb"
 
+config KVM_BOOT_EFI_STUB_RST_ATK_MITIGATION
+	bool "TCG Reset Attack Mitigation"
+	default n
```

Comment:
> Since this is not really implemented in `OVMF`'s `NVRAM` variables we disable this by default.

I don't follow, can you please elaborate on that?

Reply:
I missed this comment, but it is the same answer

> Yes, it is quite a common case. By default, default distribution installed OVMF does not have such variables enabled, but you can configure it to be if you manually build OVMF, but I doubt that is the case for your usual user. Many baremetal machines do have it though.